### DISCUSSION TITLE
Built-in visualization of GapEncoder topics?

### DISCUSSION THREAD

**Discussion by jeromedockes** [skrub-data/skrub#909] [General]
One of the strengths of the GapEncoder is its interpretability. I wonder if we could have some fitted attributes or methods to inspect the topics more easily? maybe generating a plot like the one in [example 2](https://skrub-data.org/stable/auto_examples/02_feature_interpretation_with_gapencoder.html), or per-topic bar plots, or just printing lists of term: loading lines ... or getting them as a dictionary or Series to make our own plots.

we can already see the top N terms of each topic with `get_feature_names_out`, but we may want to see the coefficient as well, or get the result in a more convenient format than a comma-separated list

**Comment by GaelVaroquaux** [comment]
> One of the strengths of the GapEncoder is its interpretability. I wonder if we could have some fitted attributes or methods to inspect the topics more easily? maybe generating a plot like the one in example 2, or per-topic bar plots,

Long term yes. Mid-term, I would like to move forward our html UX first
